file/file_test.go

```diff
@@ -429,3 +429,118 @@ func TestRemoveAll(t *testing.T) {
 	errs := RemoveAll(tmpdir)
 	require.Equal(t, 0, len(errs), "couldn't remove folder: %s", errs)
 }
+
+func createTempFile(fileContent string) (*os.File, string, error) {
+	f, err := os.CreateTemp("", "output")
+	if err != nil {
+		return nil, "", fmt.Errorf("couldn't create file: %s", err)
+	}
+	_, _ = f.Write([]byte(fileContent))
+	f.Close()
+	fileName := f.Name()
+	f, err = os.Open(fileName)
+	if err != nil {
+		return nil, "", fmt.Errorf("couldn't create file: %s", err)
+	}
+	return f, fileName, nil
+}
+
+func TestCountLineSuccess(t *testing.T) {
+	fTest, fileNameTest, err := createTempFile("test\ntest1\ntest2\ntest3\ntest4\n")
```

Comment:
Why don't you just store the files in a folder statically (see https://github.com/projectdiscovery/nuclei/tree/main/integration_tests/file/data)?